plugins/gh.ml

```diff
@@ -63,20 +64,17 @@ let reply_of_key key =
 
 let name = "Pull request linker"
 
-let rule m = m.Irc_message.command = "PRIVMSG" &&
-  match m.Irc_message.trail with
-  | Some msg -> Re.execp pullreq_re msg
-  | None -> false
+let rule = function
+  | {Irc_message.command = Irc_message.PRIVMSG (_, msg)} ->
+    Re.execp pullreq_re msg
+  | _ -> false
 
 let run ~connection ~message =
-  match message.Irc_message.trail with
-  | None -> return ()
-  | Some msg ->
+  match message.Irc_message.command with
+  | Irc_message.PRIVMSG (channel, msg) ->
     let keys = extract_pullreqs msg in
     List.map reply_of_key keys |>
     Lwt_list.iter_p (fun reply ->
-      let channels = message.Irc_message.params in
```

Comment:
Does the new library only support PRIVMSG to a single channel. My testing when writing this originally suggested it could be in many channels: https://tools.ietf.org/html/rfc2812#section-3.3.1